src/sample-addresses.js

```diff
@@ -1 +1 @@
-export const maliciousAddress = '0x5FbDB2315678afecb367f032d93F642f64180aa3';
```

Comment:
this address is no longer flagged as malicious (https://consensys.slack.com/archives/C08JQ6DDCSU/p1744401213662199?thread_ts=1744173737.539829&cid=C08JQ6DDCSU) replacing with a new address which isflagged

Reply:
the change has been reverted, as the address i working again as malicious